version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.1.3"
+ThisBuild / version := "0.1.2"
```

Comment:
It seems like this was already set to `"0.1.3"` previously. What's the reasoning for bumping it down? A little confused.

Reply:
It was the auto commit from release process, so I removed the code that sets the next release version.